2_feature_importance/2_feature_importance.ipynb

```diff
@@ -0,0 +1,500 @@
+{
```

Comment:
_Commented on notebook `2_feature_importance/2_feature_importance.ipynb` Cell 9 Line 12_
```python
use('ggplot')
fig, ax = plt.subplots()
feature_importance.plot.bar(x = "feature",
                            y = "importance",
                            ax=ax)
ax.set_title("Impurity feature importance")
ax.set_ylabel("Mean decrease in impurity")
fig.tight_layout()
```
Might be nice in the end to add a markdown not only discussing how it is obtained, but also some intuitions about how to understand/use it, and some strengths/limitations (which also can help transition to the next feature importance metrics). Same for the remaining metrics.

_Respond and view the context [here](https://gitnotebooks.com/oriol-pomarol/codegeo_workshops/pull/1)._

Reply:
This is part of the presentation.